apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -589,7 +589,7 @@ async function openLinksInParentFrame( calypsoPort ) {
 		}
 		e.preventDefault();
 		calypsoPort.postMessage( {
-			action: 'viewPost',
+			action: 'openLinkInParentFrame',
```

Comment:
Non blocking:  If we are removing/renaming the only `'viewPost'` action here, we can probably remove the `EditorActions.ViewPost` definitions and conditionals in calypsoify-iframe.tsx as well?

Reply:
I left a note in the PR description, but to elaborate, if I understand things correctly, we want to add `openLinkInParentFrame` in this PR and remove references to the [ViewPost message](https://github.com/Automattic/wp-calypso/pull/62868/files#diff-94cbcc079892f7f8fb113398707b6a9b6394766e051b1f2c28f98334ad25015fL406-L409) in a follow-up.

We handle renaming in this way because changes to `client/gutenberg/editor/calypsoify-iframe.tsx` are deployed immediately when merged to Calypso, while changes to the `wpcom-block-editor/src/calypso/features/iframe-bridge-server.js` are not. If we were to rename everything in a single PR, the stale version of `wpcom-block-editor` would reference non-existent messages in client/gutenberg/editor/calypsoify-iframe.tsx.